crates/forge/tests/e2e/running.rs

```diff
@@ -1145,9 +1141,9 @@ fn call_nonexistent_selector() {
         output,
         indoc! {r"
         Collected 1 test(s) from nonexistent_selector package
-        Running 1 test(s) from tests/
-        [PASS] nonexistent_selector_integrationtest::test_contract::test_unwrapped_call_contract_syscall (gas: ~103)
         Running 0 test(s) from src/
+        Running 1 test(s) from tests/
+        [PASS] nonexistent_selector_integrationtest::test_contract::test_unwrapped_call_contract_syscall (gas: ~[..])
```

Comment:
Maybe let's just do this?

```suggestion
        [PASS] nonexistent_selector_integrationtest::test_contract::test_unwrapped_call_contract_syscall [..]
```

Reply:
Imo can leave is as `(gas: ~[..])`, it's more readable and explicitly shows that gas' value should vary.

Reply:
Both options work for me